packages/zero-client/src/client/options.ts

```diff
@@ -118,6 +118,12 @@ export interface ZeroOptions<
 
   /**
    * `onOnlineChange` is called when the Zero instance's online status changes.
+   *
+   * @deprecated Use `onOnline` on the Zero instance instead. e.g.
```

Comment:
Nuts. @grgbkr had gone to a lot of trouble to pass in all the configuration to `Zero`. But I think didn't consider this problem/use-case. So maybe passing in all configuration cannot work.

Reply:
We can support both here. The adding/removal of listeners doesn't need to be all up-front. And this doesn't need to be deprecated.

Or, if it's better, we can move this all to the React/Solid layers. And continue to use `onOnlineChange` and track the online state with React Context.